### PR TITLE
[#150492095] Update trade confirm screen - change portfolio button by…

### PR DESCRIPTION
**Pull request by guillaumedebavelaere**
… orders status

### CODE REVIEW

File: TradeItIosTicketSDK2/TradeItSDK.swift [hunk 1]
```diff
@@ -4,7 +4,7 @@ import UIKit
     // MARK: Settable properties
 
     public static var theme: TradeItTheme = TradeItTheme.light()
-    public static var isPortfolioEnabled = true
+    public static var isOrderStatusEnabled = true
```
Comment (jsom) [inline TradeItIosTicketSDK2/TradeItSDK.swift]: Why is this removed? It seemed from the story that we want both the Portfolio and the Order Status button available but configurable.

Comment (guillaumedebavelaere) [reply TradeItIosTicketSDK2/TradeItSDK.swift]: @jsom we replaced the portfolio view button by the order view button, and the `isPortfolioEnabled ` is not used anywhere else.

Comment (jsom) [reply TradeItIosTicketSDK2/TradeItSDK.swift]: It is used by partners - they will have to modify their code to use this new boolean

Comment (jsom) [reply TradeItIosTicketSDK2/TradeItSDK.swift]: I think we should leave the flag as is but use it to also cover this case. If someone disables the portfolio, that should also mean disabling the orders so essentially it all means the same.
